src/Microsoft.AspNet.Razor/Tokenizer/Tokenizer.cs

```diff
@@ -179,9 +277,10 @@ protected StateResult AfterRazorCommentTransition()
                 // We've been moved since last time we were asked for a symbol... reset the state
                 return Transition(StartState);
             }
+
             AssertCurrent('*');
             TakeCurrent();
-            return Transition(EndSymbol(RazorCommentStarType), RazorCommentBody);
+            return Transition(1002, EndSymbol(RazorCommentStarType));
```

Comment:
I already fixed this by creating another enum. you'll see in the next iteration..